fork-runner/src/main/java/com/shazam/fork/runner/listeners/ProgressTestRunListener.java

```diff
@@ -58,7 +57,7 @@ public void testEnded(TestIdentifier test, Map<String, String> testMetrics) {
 
     @Override
     public void testRunFailed(String errorMessage) {
-        poolProgressTracker.failedTest();
```

Comment:
I was under impression that we have a limit number of retries for failed tests which also includes a case when a test run fails/crashes. Do we need to adjust that logic to also cover this new `failedTestRun` count?

Reply:
This listener only updates the test count and retries are tracked in a different place. Afaik TestRun fails will be retried by the runner based on the reports missing and normal failures will follow their old flow as normal, both for logging and for retrying. Am I missing something?

Reply:
(See RetryListener for the logic, it seems that there's some guards against double reschedule already there)